src/CAN/CANMotor.cpp

```diff
@@ -5,7 +5,11 @@
 
 #include <chrono>
 #include <cmath>
+#include <condition_variable>
+#include <mutex>
+#include <queue>
 #include <thread>
+#include <vector>
```

Comment:
Are any of these new includes required? They seem like they might have been vestigial in `CANMotor_PSOC.cpp`, so we should probably remove them.

Note that we still want `<thread>`, so leave that in.